openwisp_monitoring/monitoring/models.py

```diff
@@ -9,13 +9,15 @@ class Meta(AbstractMetric.Meta):
         swappable = swappable_setting('monitoring', 'Metric')
 
 
-class Graph(AbstractGraph):
-    class Meta(AbstractGraph.Meta):
+class Chart(AbstractChart):
+    class Meta(AbstractChart.Meta):
         abstract = False
-        swappable = swappable_setting('monitoring', 'Graph')
+        db_table = 'monitoring_chart'
```

Comment:
can you explain why setting the db_table explicitly here and in the other occurrences is needed? Can we do it without doing this?

Reply:
Django's migrations.RenameModel works great for a simple rename but it needs you to manually create the migration script and take care of few small things like renaming the table name (not much trouble and better than deleting and creating a new model, I think)
>Can we do it without doing this?

Yes, we can do the same in two approaches one is using `db_table` while other is adding a line in the migrations script. I did it some time back but was asked if it can be done without this so shifted to `db_table` https://github.com/openwisp/openwisp-monitoring/pull/112/commits/f7ea2de262133fa7e16ce424f6787dd959c0a77b#diff-5f2ccbbd9d955a5d9006eb87270c59d3L15.
I am not sure which approach is better here, please let me know.

Reply:
It's ok to have additional lines in the migration scripts.
`db_table` in the model options (`Meta` class) should be used only when dealing with legacy systems.

Reply:
Done :+1:
Can you please explain why it's reserved specifically for legacy systems, I didn't came across this earlier so might learn something. Thanks.